modules/components/src/DropDown/MultiSelectDropDown.js

```diff
@@ -0,0 +1,319 @@
+import React, { useEffect, useRef, useState } from 'react';
+import './DropDown.css';
+
+let instances = 0;
+
+const ArrowIcon = ({ isOpen }) => {
```

Comment:
Can we move SVG icons to their own dedicated files? Typically we have a folder full of svg's to reference instead of burying them in files.